--- a/Makefile
+++ b/Makefile
@@ -34,7 +34,7 @@
 GOACC_COMMIT := ddc355013f90fea78d83d3a6c71f1d37ac07ecd5

 DEPGET := cd /tmp && GO111MODULE=on go get -v
-GOBUILD := GO111MODULE=on go build -v
+GOBUILD := GO111MODULE=on $(GO) build -v
 GOINSTALL := GO111MODULE=on go install -v
 GOTEST := GO111MODULE=on go test

@@ -217,7 +217,7 @@

 vendor:
 	@$(call print, "Re-creating vendor directory.")
-	rm -r vendor/; GO111MODULE=on go mod vendor
+	rm -r vendor/; GO111MODULE=on $(GO) mod vendor

 ios: vendor mobile-rpc
 	@$(call print, "Building iOS framework ($(IOS_BUILD)).")
